.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         uses: NullVoxPopuli/action-setup-pnpm@v2
         with:
           node-version: 16.x
-          no-lockfile: true
```

Comment:
There is an issue with the `no-lockfile` boolean flag. An [issue](https://github.com/NullVoxPopuli/action-setup-pnpm/issues/16) was open. By passing directly `--no-lockfile` in the args, we have the following warning on the `pnpm` installation: "A pnpm-lock.yaml file exists. The current configuration prohibits to read or write a lockfile."